core/api/src/app/wallets/update-pending-invoices.ts

```diff
@@ -55,394 +26,26 @@ export const handleHeldInvoices = async (logger: Logger): Promise<void> => {
     ) => {
       logger.trace("updating pending invoices %s in worker %d", index)
 
-      return updateOrDeclinePendingInvoice({
-        walletInvoice,
-        logger,
-      })
+      return WalletInvoiceChecker(walletInvoice).shouldDecline()
+        ? declineHeldInvoice({ walletInvoice, logger })
+        : updatePendingInvoice({ walletInvoice, logger })
     },
   })
 
   logger.info("finish updating pending invoices")
 }
 
-export const updatePendingInvoiceByPaymentHash = async ({
+export const handleHeldInvoiceByPaymentHash = async ({
```

Comment:
Should this file be renamed to `handle-held-invoices`?

Reply:
https://github.com/GaloyMoney/galoy/pull/3582/commits/dd6f2579bae7e2200a263902e8a9ba4e1aec6fbc